xcode/Subconscious/Shared/Components/AddressBook/AddFriendView.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  AddFriendView.swift
+//  Subconscious (iOS)
+//
+//  Created by Ben Follington on 2/27/23.
+//
+
+import SwiftUI
+import ObservableStore
+
+struct AddFriendView: View {
+    var state: AddressBookModel
+    var send: (AddressBookAction) -> Void
+    
+    @Environment(\.presentationMode) var presentationMode: Binding<PresentationMode>
+    
+    @State var did: String = ""
```

Comment:
Not blocking: let's move all @State to the model. However, I do not want to hold up landing this. We can deal about this in a future PR.